pytition/petition/templates/petition/index.html

```diff
@@ -18,10 +18,18 @@ <h1 class="jumbotron-heading">{% trans "Welcome to Pytition"%}</h1>
     {% endif %}
   </div>
 </section>
-<section class="petition-list bg-light">
+<section id="petition-list-id" class="petition-list bg-light">
   <div class="text-center container">
     <h2>{% trans "View latest petitions" %}</h2>
-    {% include 'petition/generic_petition_list.html' with petitions=petitions %}
+    <div class="d-flex justify-content-end">
+      <div class="btn-group" role="group" aria-label="sort-buttons">
+          <a class="btn {% if sort == 'desc' %} btn-secondary {% else %} btn-outline-secondary {% endif %}"
+              href="{% url "index" %}?sort=desc#petition-list-id" role="button">Last Created</a>
```

Comment:
You need to use {% trans "Last Created" %} for all texts so that they can be translated